apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -75,7 +75,8 @@ defmodule AndiWeb.EditLiveView do
 
       <%= live_component(@socket, AndiWeb.EditLiveView.DeleteDatasetModal, visibility: @delete_dataset_modal_visibility, id: @dataset_id) %>
 
-      <div phx-hook="showSnackbar">
+      <div id="edit-page-snackbar" phx-hook="showSnackbar">
+        <div style="display: none;"><%= @click_id %></div>
```

Comment:
This click id is just a random uuid that updates any time the save button is pressed or the snackbar should be triggered. This solves the case when the snackbar wouldn't show if nothing in this div was updated. Kind of a hacky solution